2_Collections.py

```diff
@@ -0,0 +1,77 @@
+# Home task_2 Collections
+"""
+Write a code, which will:
+
+1. create a list of random number of dicts (from 2 to 10)
+dict's random numbers of keys should be letter,
+dict's values should be a number (0-100),
+example:[{'a': 5, 'b': 7, 'g': 11}, {'a': 3, 'c': 35, 'g': 42}]
+
+2. get previously generated list of dicts and create one common dict:
+if dicts have same key, we will take max value, and rename key with dict number with max value
+if key is only in one dict - take it as is,
+example:{'a_1': 5, 'b': 7, 'c': 35, 'g_2': 42}
+Each line of code should be commented with description.
+
+Commit script to git repository and provide link as home task result.
+"""
+
+# Task_1 Create list of 100 random numbers from 0 to 1000
+import random
+import string
+
+# Generate random number of list elements in the range from 2 to 10
+cnt_elements = random.randint(2, 10)
+
+dict_list = []
+
+# Create list of  dictionaries
+for i in range(cnt_elements):
+    # Generate number of elements in the list item. Each dictionary can contain from 2 to 10 elements
+    n = random.randint(2, 10)
+    # Initiate dictionary element
+    dict_elem = {}
```

Comment:
Try also to use list comprehensions and dict comprehensions. You will see such code in many solutions.
https://realpython.com/list-comprehension-python/